pkg/engine/adapters/rclient.go

```diff
@@ -43,12 +47,18 @@ func (a *rclientAdapter) ForRef(ctx context.Context, ref string) (*engineapi.Ima
 	if err != nil {
 		return nil, fmt.Errorf("failed to fetch config for image reference: %s, error: %v", ref, err)
 	}
+	rawIndex, err := imageIndex.RawManifest()
+	if err != nil {
+		return nil, fmt.Errorf("failed to fetch image index for image reference: %s, error: %v", ref, err)
+	}
+
 	data := engineapi.ImageData{
 		Image:         ref,
 		ResolvedImage: fmt.Sprintf("%s@%s", parsedRef.Context().Name(), desc.Digest.String()),
 		Registry:      parsedRef.Context().RegistryStr(),
 		Repository:    parsedRef.Context().RepositoryStr(),
 		Identifier:    parsedRef.Identifier(),
+		ImageIndex:    rawIndex,
```

Comment:
Image index might not be the best name for this as it is a manifest list, i dont think imageindex explains the field's content properly

Reply:
OCI calls it "Image Index" while Docker calls it "Manifest List", should we use Docker's version?